src/stk500v2.c

```diff
@@ -3710,6 +3750,20 @@ static void stk500v2_print_parms1(const PROGRAMMER *pgm, const char *p, FILE *fp
         fmsg_out(fp, "%.3f %s\n", f, unit);
       }
     }
+    fmsg_out(fp, "%sSCK period      : %.1f us\n", p,
+	    stk500v2_sck_to_us(pgm, sck_duration));
+
+  const char *unit;
+  double f = PDATA(pgm)->xtal;
+  if (f >= 1e6) {
+    f /= 1e6;
+    unit = "MHz";
+  } else if (f >= 1e3) {
+    f /= 1000;
+    unit = "kHz";
+  } else
+    unit = "Hz";
+  fmsg_out(fp, "%sXTAL frequency  : %.3f %s\n", p, f, unit);
```

Comment:
No space before colon

Reply:
He may have added that space so that all colons lines up when running the `parms` command in terminal mode.

Reply:
Of course, I missed that. Thanks for reviewing the review